### ISSUE TITLE
Only show status for group, not all public services

### ISSUE THREAD

**Issue by ilyakonovalenko** [statping-ng/statping-ng#183]
It would be nice to only show status for group (all online, partially online, all offline), not all visible services.

Such scenario would be useful in case you don't want to show all services names and detailed services stats to non-admins, but do want to show aggregated status for all services in group.

**Comment by lraikhman** [comment]
Agreed. I would like to monitor 300+ routers via a call to their public cloud control API.
These are group-able by customer.
I'd love the front page to default to expandable set of groups with traffic lights Red and Green listing number of services online and offline.
